mungegithub/mungers/sync/issue-sync.go

```diff
@@ -201,10 +204,16 @@ func (s *IssueSyncer) createIssue(source IssueSource) (issueNumber int, err erro
 		panic(fmt.Errorf("Programmer error: %v does not contain %v!", body, id))
 	}
 
+	owner := ""
+	if s.owner != nil {
```

Comment:
Suggest never letting this be nil? Or document that it can be nil in the struct.